owncloud/indico_owncloud/templates/owncloud_filepicker_widget.html

```diff
@@ -12,6 +12,7 @@
 
 {% block javascript %}
     <script>
+        document.querySelector('input[type=submit]').disabled = true;
```

Comment:
This is still way too generic ;/ It'll disable submit inputs on the whole page (even though there may not be any others).

```suggestion
        document.querySelector('#attachment-owncloudfilepicker-form input[type=submit]').disabled = true;
```